napalm/base/netmiko_helpers.py

```diff
@@ -0,0 +1,47 @@
+# The contents of this file are licensed under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with the
+# License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+from __future__ import unicode_literals
+import inspect
+from netmiko import BaseConnection
+
+
+def netmiko_args(optional_args):
+    """Check for Netmiko arguments that were passed in as NAPALM optional arguments.
```

Comment:
Can you confirm the optional_args documentation (RTD) for netmiko based drivers are in synch with what this supports?

Reply:
I will update the documentation after this is merged. I expect the documentation is wrong, but I am pretty sure it was wrong both before and after this.

This will make the documentation easier since it will be consistent across all three drivers and will support all Netmiko `__init__` arguments.

@dbarrosop